### ISSUE TITLE
report error(such as dependency dll can't be found) in the dry run mode?

### ISSUE THREAD

**Issue by asmwarrior** [brechtsanders/pedeps#16]
Hi, when I run the command:

~~~
F:\code\codeblocks64-abc\lib\codeblocks\plugins>..\..\..\bin\copypedeps.exe -r -d -v libhelp_plugin.dll ..\..\..\bin
~~~

I got the result like below:
~~~
libhelp_plugin.dll -> ..\..\..\bin\libhelp_plugin.dll
~~~

This means `libhelp_plugin.dll` will be copied to the destination folder.

But in-fact, the dependency dlls of this `libhelp_plugin.dll` are **not** found in the PATH, so I think we need to find a way to report the missing dll errors?

Thanks.

**Comment by brechtsanders** [comment]
Good point. It should at least be properly documented that `copypedeps` looks in the `PATH` to determine what to copy, and there should be proper errors.

The intial goal was to have pedeps as a library and maybe the accompanying utilities were written a bit too simple.
